### PR TITLE
feat(feishu-web): add feishu connector

### PR DESCRIPTION
**Pull request by StringKe**
## Summary

add feishu connector
https://open.feishu.cn/document/common-capabilities/sso/web-application-sso/web-app-overview

## Testing

local test

### CODE REVIEW

File: packages/connector-feishu-web/package.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@logto/connector-feishu-web",
+  "version": "1.0.0-beta.12",
+  "description": "Alipay implementation.",
```
Comment (charIeszhao) [inline packages/connector-feishu-web/package.json:4]:
```suggestion
  "description": "Feishu connector.",
```

File: packages/connector-feishu-web/src/constant.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+export const defaultMetadata: ConnectorMetadata = {
+  id: 'feishu-web',
+  target: 'feishu',
+  platform: ConnectorPlatform.Web,
+  name: {
+    en: 'Feishu',
+    'zh-CN': '飞书',
+    'tr-TR': 'Feishu',
+    ko: 'Feishu',
```
Comment (charIeszhao) [inline packages/connector-feishu-web/src/constant.ts:18]: These two lines can be saved as well, as the name will fallback to English if a local translation is not found

File: packages/connector-feishu-web/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    metadata: defaultMetadata,
+    type: ConnectorType.Social,
+    platform: ConnectorPlatform.Web,
+    configGuard: feishuConfigGuard,
+    getAuthorizationUri: getAuthorizationUri(getConfig),
+    getUserInfo: getUserInfo(getConfig),
+  };
+};
+
+export default createAlipayConnector;
```
Comment (charIeszhao) [inline packages/connector-feishu-web/src/index.ts:192]:
```suggestion
const createFeishuConnector: CreateConnector<SocialConnector> = async ({ getConfig }) => {
  return {
    metadata: defaultMetadata,
    type: ConnectorType.Social,
    platform: ConnectorPlatform.Web,
    configGuard: feishuConfigGuard,
    getAuthorizationUri: getAuthorizationUri(getConfig),
    getUserInfo: getUserInfo(getConfig),
  };
};

export default createFeishuConnector;
```

**Comment by charIeszhao** [comment]
Overall looks good! Thank you for your contribution, @StringKe.
We'll help work on the rest of it, like replacing the icon, writing readme, etc. Nice work!

**Review by charIeszhao — APPROVED** [review]
Please remove the `CHANGELOG.md`

File: packages/connector-feishu-web/src/index.test.ts [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+import { mockedFeishuConfig } from './mock';
+
+const getConfig = jest.fn().mockResolvedValue(mockedFeishuConfig);
+
+describe('getAuthorizationUri', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should build authorization uri ', function () {
```
Comment (charIeszhao) [inline packages/connector-feishu-web/src/index.test.ts:16]:
```suggestion
  it('should build authorization uri', function () {
```

**Comment by StringKe** [comment]
> Please remove the `CHANGELOG.md`

Removed

File: packages/connector-feishu-web/package.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  "devDependencies": {
+    "@jest/types": "^28.1.3",
+    "@shopify/jest-koa-mocks": "^5.0.0",
+    "@silverhand/eslint-config": "1.3.0",
+    "@silverhand/ts-config": "1.2.1",
+    "@types/jest": "^28.1.6",
+    "@types/lodash.pick": "^4.4.6",
+    "@types/node": "^16.3.1",
+    "@types/supertest": "^2.0.11",
+    "@vercel/ncc": "^0.34.0",
```
Comment (charIeszhao) [inline packages/connector-feishu-web/package.json:42]:
```suggestion
    "@vercel/ncc": "^0.36.0",
```

**Review by charIeszhao — COMMENTED** [review]
The code looks OK to me now. But let's hold it until the README is added.

**Comment by wangsijie** [comment]
@StringKe What's the status of README?

**Review by darcyYe — APPROVED** [review]

File: packages/connector-feishu-web/README.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# 飞书网页
+
+先进企业协作与管理平台，一站式无缝办公协作，团队上下对齐目标，全面激活组织和个人。
+
+## 开始上手
+
+飞书网页连接器是为桌面网页应用设计的。它采用了支付宝的 OAuth 2.0 认证流程。
```
Comment (wangsijie) [inline packages/connector-feishu-web/README.md:7]: unexpected `支付宝`?

**Review by wangsijie — APPROVED** [review]
Overall LGTM

File: packages/connector-feishu-web/README.md (hunk 6 above)
Comment (darcyYe) [reply packages/connector-feishu-web/README.md]: Fixed.

**Pull request merged**